docs/ansible/roles/debops.resources/defaults-detailed.rst

```diff
@@ -59,10 +59,9 @@ are performed; this means that you can put a file in the :file:`by-group/all/`
 directory and then override it using a host-specific directory.
 The groups directories are read in the order dictated by Ansible during inventory parsing.
 
-See `Ansible - Playbooks Varibles<https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#accessing-information-about-other-hosts-with-magic-variables>`_
-to learn about the variable `group_names`.
-See `Ansible - Working with Inventory<https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#how-variables-are-merged>`_
-for more information on how to use `ansible_group_priority` to change the merge order
+See .. _`Ansible - Playbooks Variables`: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#accessing-information-about-other-hosts-with-magic-variables
+to learn about the ``group_names`` variable, and .. _`Ansible - Working with Inventory`: https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#how-variables-are-merged
```

Comment:
No, this is now completely mangled. The anonymous references in reStructuredText are in the form:
```rst
Part of the text with `anonymous reference`__ and the end.

.. __: https://anonymous-link/
```
Remember to use two underscores for anonymous references. The URLs are specified after the references in a separate line each, and the order of the references is significant.

Reply:
I used anonymous references but the make file keep complaining about a missing third reference. Then I changed it to the current code.